group_vars/pre_install_toolkit/packages.suse.yml

```diff
@@ -29,10 +29,10 @@ packages:
   - wol=0.7.1-2.5
   # CSM
   - canu=1.8.1-1
-  - cray-site-init=1.32.6-1
+  - cray-site-init=1.32.7-1
   - ilorest=4.2.0.0-20
   - metal-basecamp=1.2.7-1
-  - metal-ipxe=2.4.8-1
+  - metal-ipxe=2.5.1-1
```

Comment:
I think we need to keep it below 2.5.0 for the PIT. 2.5 is where the script changed for Fawkes.

Reply:
Ah, right. I think I made that mistake once before.